site/docs/developer_documentation/getting_started.md

```diff
@@ -40,6 +44,42 @@ export LIBINT_DATA_PATH=${CONDA_PREFIX}/share/libint/2.3.1/basis
 
 You will have to either export this environment variable every time you activate the `gqcp` environment or (better) put this export in your .bashrc or (preferred) [add this environment variable to your virtual environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#saving-environment-variables).
 
+### Docker installation
+
+First, [install Docker](https://docs.docker.com/get-docker/). VSCode [has excellent support for containers](https://code.visualstudio.com/docs/remote/containers-tutorial). Using the [Visual Studio Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers) you can mount the directory in which you have cloned the GQCP repo in the Docker container. There are two ways to get the Docker container up and running:
+
+1. You can let VSCode [build the image on your local machine](https://code.visualstudio.com/docs/remote/containers#_quick-start-open-an-existing-folder-in-a-container). 
+2. You can use a prebuilt container by pulling the `gqcp-dev` image from our organization 
```

Comment:
I can read that there is now a difference between the Conda environment and the Docker container name.

Could it be possible to name these consistently? It doesn't really matter which name we choose, but I'm personally leaning towards `gqcp-dev`.

Reply:
I agree with `gqcp-dev`. In light of the recent Docker support, we will discuss in a future meeting whether `gqcp-dev` Conda support is still required.